### PR TITLE
[bitnami/airflow] feat: 🔒 Enable networkPolicy

### PR DESCRIPTION
**Pull request by alvneiayu**
### Description of the change

This PR normalizes the use of NetworkPolicy in the chart. Adds all Bitnami standards for NetworkPolicies as well as enabling it by default, in order to comply with security checklists.

### Benefits

More security in the chart

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/airflow/templates/scheduler/networkpolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      to:
+        - podSelector:
+            matchLabels: {{- include "common.labels.matchLabels" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 14 }}
+    {{- if .Values.scheduler.networkPolicy.extraEgress }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.scheduler.networkPolicy.extraEgress "context" $ ) | nindent 4 }}
+    {{- end }}
+  ingress:
+    - ports:
+        - port: {{ .Values.service.ports.http }}
+        - port: {{ .Values.worker.containerPorts.http }}
```
Comment (FraPazGal) [inline bitnami/airflow/templates/scheduler/networkpolicy.yaml:45]: Same as above:

```suggestion
        - port: {{ .Values.scheduler.containerPorts.http }}
```

**Review by FraPazGal — COMMENTED** [review]
Thanks for the PR @alvneiayu! I left a few comments as a review, please check them whenever you can

File: bitnami/airflow/templates/worker/networkpolicy.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    {{- end }}
+  ingress:
+    - ports:
+        - port: {{ .Values.service.ports.http }}
+        - port: {{ .Values.worker.containerPorts.http }}
+      {{- if not .Values.worker.networkPolicy.allowExternal }}
+      from:
+        - podSelector:
+            matchLabels:
+              {{ template "appsmith.worker.fullname" . }}-client: "true"
```
[the remark this answers is not in the document]
Comment (alvneiayu) [reply bitnami/airflow/templates/worker/networkpolicy.yaml]: fixed, good catch ^^ thanks a lot for your review

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**